KarhooUISDK/UIComponents/AddressBarMVP/PrebookFieldMVP/KarhooPrebookFieldView.swift

```diff
@@ -16,7 +16,16 @@ public struct KHPrebookFieldID {
 }
 
 public final class KarhooPrebookFieldView: UIView {
-    
+
+    // MARK: - Nested types
+
+    private enum Constants {
```

Comment:
Could we use values from the UIConstants here instead? I was thinking about using the standard 44. Or is there another reason to use 35?